package.json

```diff
@@ -29,6 +29,8 @@
     "typescript": "~4.4.2"
   },
   "dependencies": {
+    "@aws-cdk/aws-lambda-event-sources": "^1.124.0",
+    "@aws-sdk/client-sqs": "^3.34.0",
```

Comment:
These should not be added at the root level.

Reply:
Fixed this.